### PR TITLE
[docs] Re-org topics in Admin API section

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation
The sub-commands for persistent is changed into "topics". And persistent topics can be partitioned and non-partitioned topics.

### Modifications
1. Refactor the topic structure, will keep only one md file for the topic in the "Admin API" section.
2. Refine the commands used in the topic section from "persistent" into "topics" for both `pulsar-admin`, `REST API` and Java.
3. Adopt switch tab for `pulsar-admin`, `REST API` and Java methods.

### Follow-ups
1. Refine the partitioned and non-partitioned sections.
2. Remove the other 3 files and check whether there is any link to the 3 deleted files throughout.
3. Check the md ID refinement.
4. Fix the REST API links in https://github.com/apache/pulsar/issues/8377

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
The whole PR will be a huge PR, I can add "partitioned" and "non-partitioned topics" later in another PR.
Please double check the REST API in the "Last Message Id" section.
The original one is "{% endpoint Get /admin/v2/topics/:tenant/:namespace/:topic/lastMessageId %}".
I've changed it into "{@inject: endpoint|Get|/admin/v2/topics/:tenant/:namespace/:topic/lastMessageId}".

**Comment by Jennifer88huang** [comment]
All the REST API links do not work, open an issue in https://github.com/apache/pulsar/issues/8377

**Comment by Jennifer88huang** [comment]
@zymap could you please also help check the update? thank you.

File: site2/docs/admin-api-persistent-topics.md [hunk 1]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
-admin.persistentTopics().unload(topic);
+admin.topics().unload(topic);
 ```
 
+<!--END_DOCUSAURUS_CODE_TABS-->
+
 ### Get stats
 
-It shows current statistics of a given non-partitioned topic.
+You can view the following statistics of a given non-partitioned topic.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:172]:
```suggestion
You can check the following statistics of a given non-partitioned topic.
```
check through the whole updates

File: site2/docs/admin-api-persistent-topics.md [hunk 2]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
+  -   **msgThroughputIn**: The sum of all local and replication publishers' publish rates in bytes per second.
 
-  -   **msgRateOut**: The sum of all local and replication consumers' dispatch rates in messages per second
+  -   **msgRateOut**: The sum of all local and replication consumers' dispatch rates in messages per second.
 
-  -   **msgThroughputOut**: Same as above, but in bytes per second instead of messages per second
+  -   **msgThroughputOut**: The sum of all local and replication consumers' dispatch rates in bytes per second.
 
-  -   **averageMsgSize**: The average size in bytes of messages published within the last interval
+  -   **averageMsgSize**: The average size(in bytes) of messages published within the last interval.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:182]:
```suggestion
  -   **averageMsgSize**: The average size (in bytes) of messages published within the last interval.
```

File: site2/docs/admin-api-persistent-topics.md [hunk 3]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
+      -   **connectedSince**: The timestamp when this producer is created or reconnected last time.
 
-  -   **subscriptions**: The list of all local subscriptions to the topic
+  -   **subscriptions**: The list of all local subscriptions to the topic.
 
-      -   **my-subscription**: The name of this subscription (client defined)
+      -   **my-subscription**: The name of this subscription. It is defined by the client.
 
-          -   **msgRateOut**: Total rate of messages delivered on this subscription (msg/s)
+          -   **msgRateOut**: The total rate of messages delivered on this subscription (msg/s).
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:206]: use the same expression: use "msg/s" and (bytes/s) or in use "in messages per second" "in bytes per second"

File: site2/docs/admin-api-persistent-topics.md [hunk 4]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
+                -   **msgRateOut**: The total rate of messages delivered to the consumer (msg/s).
 
-                -   **msgThroughputOut**: Total throughput delivered to the consumer (bytes/s)
+                -   **msgThroughputOut**: The total throughput delivered to the consumer (bytes/s).
 
-                -   **consumerName**: Internal identifier for this consumer, generated by the client library
+                -   **consumerName**: The internal identifier for this consumer, generated by the client library.
 
-                -   **availablePermits**: The number of messages this consumer has space for in the client library's listen queue. A value of 0 means the client library's queue is full and receive() isn't being called. A nonzero value means this consumer is ready to be dispatched messages.
+                -   **availablePermits**: The number of messages this consumer has space for in the client library's listen queue. `0` means the client library's queue is full and `receive()` isn't being called. A nonzero value means this consumer is ready to be dispatched messages.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:232]:
```suggestion
                -   **availablePermits**: The number of messages that the consumer has space for in the client library's listen queue. `0` means the client library's queue is full and `receive()` isn't being called. A non-zero value means this consumer is ready for dispatched messages.
```

File: site2/docs/admin-api-persistent-topics.md [hunk 5]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
 
-                -   **unackedMessages**: Number of unacknowledged messages for the consumer
+                -   **unackedMessages**: The number of unacknowledged messages for the consumer.
 
-                -   **blockedConsumerOnUnackedMsgs**: Flag to verify if the consumer is blocked due to reaching threshold of unacked messages
+                -   **blockedConsumerOnUnackedMsgs**: The flag to verify if the consumer is blocked due to reaching threshold of the unacked messages.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:236]:
```suggestion
                -   **blockedConsumerOnUnackedMsgs**: The flag used to verify if the consumer is blocked due to reaching threshold of the unacknowledged messages.
```

File: site2/docs/admin-api-persistent-topics.md [hunk 6]
```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
...
                 
-                -   **lastConsumedTimestamp**: The timestamp of the consumer last consume a message
+                -   **lastConsumedTimestamp**: The timestamp when the consumer reads a message the last time. 
           
-                -   **lastAckedTimestamp**: The timestamp of the consumer last ack a message
+                -   **lastAckedTimestamp**: The timestamp when the consumer acks a message the last time.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:240]:
```suggestion
                -   **lastAckedTimestamp**: The timestamp when the consumer acknowledges a message the last time.
                
```

File: site2/docs/admin-api-persistent-topics.md [hunk 7]
```diff
@@ -307,94 +299,95 @@ It shows current statistics of a given non-partitioned topic.
...
+  -   **currentLedgerSize**: The size in bytes of messages written to the ledger that is currently open for writing.
 
-  -   **lastLedgerCreatedTimestamp**: time when last ledger was created
+  -   **lastLedgerCreatedTimestamp**: The time when the last ledger is created.
 
-  -   **lastLedgerCreationFailureTimestamp:** time when last ledger was failed
+  -   **lastLedgerCreationFailureTimestamp:** The time when the last ledger failed.
 
-  -   **waitingCursorsCount**: How many cursors are "caught up" and waiting for a new message to be published
+  -   **waitingCursorsCount**: How many cursors are "caught up" and waiting for a new message to be published.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:340]: The number of cursors that are ....?

File: site2/docs/admin-api-persistent-topics.md [hunk 8]
```diff
@@ -307,94 +299,95 @@ It shows current statistics of a given non-partitioned topic.
...
-      -   **size**: Size of messages written to this ledger (in bytes)
+      -   **size**: The size of messages written to this ledger (in bytes).
 
-      -   **offloaded**: Whether this ledger is offloaded
+      -   **offloaded**: Whether this ledger is offloaded.
 
   -   **compactedLedger**: The ledgers holding un-acked messages after topic compaction.
  
-      -   **ledgerId**: Id of this ledger
+      -   **ledgerId**: The Id of this ledger.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-persistent-topics.md:360]:
```suggestion
      -   **ledgerId**: The ID of this ledger.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments.

File: site2/docs/admin-api-persistent-topics.md [hunk 9]
```diff
@@ -1,80 +1,78 @@
...
   my-tenant/my-namespace
 ```
 
-#### REST API
-
-{@inject: endpoint|GET|/admin/v2/persistent/:tenant/:namespace|operation/getList}
-
-#### Java
+<!--REST API-->
+{@inject: endpoint|GET|/admin/v2/topics/:tenant/:namespace|operation/getList}
```
Comment (zymap) [inline site2/docs/admin-api-persistent-topics.md:34]: I don't think we have the `topics` endpoint to get all the topics. Actually, there have two steps on the client-side, the client will get the persistent topics and non-persistent topics then combine the result at the client-side.
I am not sure how to add two REST API endpoints. @tuteng Any ideas?

File: site2/docs/admin-api-persistent-topics.md [hunk 10]
```diff
@@ -1,80 +1,78 @@
...
-
 ```
 
-#### REST API
-
-{@inject: endpoint|POST|/admin/v2/persistent/:tenant/:namespace/:topic/permissions/:role|operation/grantPermissionsOnTopic}
-
-#### Java
+<!--REST API-->
+{@inject: endpoint|POST|/admin/v2/topics/:tenant/:namespace/:topic/permissions/:role|operation/grantPermissionsOnTopic}
```
Comment (zymap) [inline site2/docs/admin-api-persistent-topics.md:56]: similar issue with the previous comments. We don't have the `topics` endpoint. All the following REST API endpoint has the same issue.

File: site2/docs/admin-api-persistent-topics.md (hunk 9 above)
Comment (tuteng) [reply site2/docs/admin-api-persistent-topics.md:34]: Please refer to `NON-PERSISTENT TOPIC` http://pulsar.apache.org/admin-rest-api/?version=2.6.1&apiversion=v2#operation/getList and `PERSISTENT TOPIC` http://pulsar.apache.org/admin-rest-api/?version=2.6.1&apiversion=v2#operation/getList

Comment (Jennifer88huang) [reply site2/docs/admin-api-persistent-topics.md:34]: {@inject: endpoint|GET|/admin/v2/:schema/:tenant/:namespace|operation/getList}

**Comment by Jennifer88huang** [comment]
@tuteng @zymap Thank you very much for your comments. I've updated the REST API part, could you please double confirm? Thank you.

**Review by tuteng — APPROVED** [review]

**Pull request merged**